device-types/HPE/Aruba-ec-m-h.yaml

```diff
@@ -0,0 +1,66 @@
+---
+manufacturer: HPE
+model: Aruba ec-m-h
+slug: hpe-aruba-ec-m-h
+part_number: JZ872A
+u_height: 1
+is_full_depth: true
+airflow: front-to-rear
+console-ports:
+  - name: console
+    type: rj-45
+    label: console
+power-ports:
+  - name: power0
+    type: iec-60320-c14
+    maximum_draw: 162
+    label: power0
+  - name: power1
+    type: iec-60320-c14
+    maximum_draw: 162
+    label: power1
```

Comment:
The PSUs are just exchangeable in case of failure. 
In my understanding this is not modular since both are required and there is just the option of replacing a faulty one with exactly the same model.

Reply:
They are absolutely modular. If you can remove the PSU, it is modular by definition, regardless if the device requires both to be installed. Thus these need to be modelled as `module-bays` and the applicable PSUs need to be added to the module-types folder if they do not already exist